development_scripts/reader_testers/test_ixdat_csv_reader.py

```diff
@@ -0,0 +1,23 @@
+from ixdat import Measurement
+
+
+if False:  # test the version that's online on the tutorials page
```

Comment:
These scripts are only for developer use while they're writing the reader. So that would be overkill for me since I run them from an IDE. Other developers may find this useful, so will add a comment linking to this suggestion.